datatypes.md

```
@@ -102,6 +102,131 @@ Since we cannot reason broadly about unknown values, `null` is an important part
 
 Unlike all other scalars, `null` cannot be stored as a property value.
 
+## Temporal Types
+FalkorDB supports the following temporal types that allow modeling and querying time-related data:
+
+## 1. [Date](#Date)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
Expected: #date; Actual: #Date

(MD051, link-fragments)